docs/eventlisteners.md

```diff
@@ -1,12 +1,78 @@
 # EventListener
 
-`EventListener`s connect `TriggerBinding`s to `TriggerTemplate`s and provide an
-addressable endpoint, which is where webhooks/events are directed. This is also
-where the service account is connected, which specifies what permissions the
-resources will be created (or at least attempted) with. The service account must
-have the following role bound.
+EventListener is a Kubernetes custom resource that allows users a declarative
+way to process incoming HTTP based events with JSON payloads. EventListeners
+expose an addressable "Sink" to which incoming events are directed to. Users can
+declare [TriggerBindings](./triggerbindings.md) to extract fields from events,
+and apply them to [TriggerTemplates](./triggertemplates.md) in order to create
+Tekton resources. In addition, event listeners allow lightweight event
+processing using [EventInterceptors](#event-Interceptors)
+
+---
```

Comment:
nit: I personally think the horizontal line break looks out of place. It weirdly isolates the "Table of Contents" links below since they don't have a header. I think we can probably get rid of the horizontal line break (`---`). Feel free to ignore this, as it is entirely subjective 😄

Reply:
done